web/src/lib.rs

```diff
@@ -22,6 +25,7 @@ pub(crate) mod extractors;
 mod router;
 
 pub async fn init_server(app_state: AppState) -> Result<()> {
+    info!("Connecting to DB with URI: {}", app_state.config.database_uri());
```

Comment:
I wonder if it would make sense to change this to `debug!` so we don't log the URI in production? Thinking more of down the line if we are using like an AWS database. Not really sure if the URI itself would contain any sensitive information but just a thought.

Reply:
Great question, I actually had it as debug originally but I messed up the order in my mind. I'm going to switch this to TRACE so that it only shows up when we *really* want it to. And at some point we may even decide that this is not even necessary.